ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift

```diff
@@ -346,9 +346,8 @@ public class PackageInstallViewController<Resource: LanguageResource>: UIViewCon
 
     let dismissalBlock = {
       if let nvc = self.navigationController {
-        self.dismiss(animated: true) {
-          nvc.popToRootViewController(animated: true)
-        }
+        self.dismiss(animated: true)
+        nvc.popToRootViewController(animated: false)
```

Comment:
Why the change away from a closure?

Reply:
Note the other difference:  `animated: true` -> `animated: true` is now `animated: true` -> `animated: false`.

We're cutting out the second 'animation' so running the second dismissal concurrently with the first dismissal allows the first's animation to (visually) act for both dismissals simultaneously.  This also reduces the user's wait time.  If the closure pattern was preserved, there would still be a second animation... just abbreviated.

I'd noticed in testing that there was an awkward wait while all the view shuffling occurred, so I figured it best to reduce that if possible.  I wasn't able to remove it all, but this spot was a possible optimization.